table-api/src/main/java/io/deephaven/api/agg/spec/AggSpec.java

```diff
@@ -20,150 +20,224 @@
 public interface AggSpec {
 
     /**
+     * Produces a single-row table with the absolute some of each column.
```

Comment:
This is coming from `TableOperations.absSumBy()`, and isn't really accurate for the `AggSpec`.